akka-projection-r2dbc/src/it/scala/akka/projection/r2dbc/R2dbcProjectionSpec.scala

```diff
@@ -329,7 +329,9 @@ class R2dbcProjectionSpec
       projectionTestKit.run(projectionFailing) {
         projectedValueShouldBe("e1|e2|e3|e4|e5")
       }
-      offsetShouldBe(6L)
+      eventually {
+        offsetShouldBe(6L)
+      }
```

Comment:
even though these tests are exacly-once the offset might not have been stored yet, we have only seen "e5"